plugins/modules/manifest.py

```diff
@@ -57,7 +57,7 @@
 
 EXAMPLES = '''
 - name: "Upload the RHEL developer edition manifest"
-  katello_manifest:
+  manifest:
```

Comment:
I haven't got a good suggestion, but I feel like with `katello_manifest` it was clear what the difference was between this and `redhat_manifest`. But with `subscription_manifest` we are taking a step back in terms of clarity. Not a massive step, so its probably okay, but in case anyone has other suggestions...

Reply:
Well, you can have k_manifest if you want... ;)
But if we want to remove project names sub_manifest is the clearest.

Reply:
I went with sub_man for now.